Content: New Hai advanced start (#7721)
endless-sky/endless-sky: data/human/human missions.txt

		conversation
			`	A young man carrying luggage approaches you. "You look like a knowledgeable captain," he says. "In all your travels, have you ever heard tell of any... intelligent rodents?"`
			choice

> 				`	"'Heard tell'? That's where I come from!"`

Reviewer: ```suggestion
				`	"'Heard tell'? I'm from Hai space!"`
```

			`	"Oh, never mind then," he says, with a note of disappointment in his voice. You feel good knowing that you are helping to keep the Hai's existence a secret in human space. Better safe than sorry.`
				decline
			
			label origin

> 			`	He seems shocked, "You too? You're not joking with me, are you? I need to go to <destination>, and I can afford to give you <payment> for it."`

Reviewer: ```suggestion
			`	He seems shocked. "You too? You're not joking with me, are you? I need to go to <destination>, and I can afford to give you <payment> for it."`
```

				goto choice
			label hai
			`	"If that's the case, could you bring me to <destination>? I'll give you <payment>."`
			label choice
			choice
				`	"<planet>? I come from there!"`
					to display
						has "hai space start"
				`	"Alright, I'll take you there."`
					accept
				`	"Sorry, I'm too busy right now."`
					decline
				`	"Why would you want to go all the way out to <planet>?"`
					goto explain

> 			`	"No way, you as well?" He stops for a second to process what you just said, "They say the universe is a small place, but I never believed any of that, until now, that is. I grew up in the city of Darwin."`

Reviewer: ```suggestion
			`	"No way, you as well?" He stops for a second to process what you just said. "They say the universe is a small place, but I never believed any of that. Until now, that is. I grew up in the city of Darwin."`
```

> 			`	"Me too," you reply. "Almost felt like human space, we did not even have to learn hai. And those fluffy rodents always have a way to make you feel at home!"`

Reviewer 2: ```suggestion
			`	"Me too," you reply. "Almost felt like human space; we did not even have to learn Hai. And those fluffy rodents always have a way to make you feel at home!"`
```

> 			`	"Me too," you reply. "Almost felt like human space; we did not even have to learn Hai. And those fluffy rodents always have a way to make you feel at home!"`

Author: ```suggestion
			`	You grew up there as well, but before you can say anything, he continues, "Almost felt like human space; we did not even have to learn Hai. And those fluffy rodents always have a way to make you feel at home!"`
```

> 			`	"Me too," you reply. "Almost felt like human space; we did not even have to learn Hai. And those fluffy rodents always have a way to make you feel at home!"`

Reviewer: I think the first bit is unnecessary, how about:
```suggestion
			`	Before you can say anything, he continues. "Almost felt like human space; we didn't even have to learn Hai. And those fluffy rodents always have a way to make you feel at home!"`
```

> 			`	Before you can say anything, he continues. "Almost felt like human space; we didn't even have to learn Hai. And those fluffy rodents always have a way to make you feel at home!"`

Reviewer: ```suggestion
			`	Before you can say anything, he continues. "Almost felt like human space; I didn't even have to learn Hai." He smiles. "They're a very welcoming species. Partially why I want to go back."`
```

			choice
				`	(Leave.)`
					decline
				`	"May I come speak with them?"`
					to display
						has "hai space start"
					goto haistart
				`	"That's your family?"`
			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
			choice
				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`

> 				`	"No problem, I hope you enjoyed the trip."`

Author: ~~I had no idea, always pictured him a human~~
I think I made it so you get a different option?

> 				`	"No problem, I hope you enjoyed the trip."`

Reviewer 3: I mean, it doesn't make sense if you later say you're half adopted by a Hai, to wonder whether Hai adoptions happen. I think it says that your adoptive father is a Hai later in this PR.

			label end
			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`
				decline
			label haistart

> 			`	"Of course, you did bring me all the way back here, after all." These are my parents, Yeerto and Taepip. His father Yeerto then speaks, "We heard about your travel agency, even went there at some point when we were thinking of relocating to human space. It ended up being a touch too expensive, though, so Elliot arranged a merchant to bring him on the way."`

Reviewer: ```suggestion
			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then speaks, "We heard about your travel agency, even went there at some point when we were thinking of relocating to human space. It ended up being a touch too expensive, though, so Benjamin arranged a merchant to bring him on the way."`
```

> 			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then speaks, "We heard about your travel agency, even went there at some point when we were thinking of relocating to human space. It ended up being a touch too expensive, though, so Benjamin arranged a merchant to bring him on the way."`

Reviewer: ```suggestion
			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then speaks, "We heard about your travel agency, even went there at some point when we were thinking of relocating to human space. It ended up being a touch too expensive, though, so Benjamin arranged for a merchant to bring him on the way."`
```

> 			`	"It feels nice to have my hard work be recognized," you answer.`

Author: ~~did I pull a MZ on myself?~~
hold on, imma make this a choice with only one option
~~you WILL say this~~ (jking, I'm guessing that isnt that good either haha)

> 			`	"It feels nice to have my hard work be recognized," you answer.`

Author: ```suggestion
			`	"Funny we almost found each others then," he adds, with a smile on his face. You smile back, and nod.`
```

> 			`	"It feels nice to have my hard work be recognized," you answer.`

Reviewer: ```suggestion
			`	"Funny we almost found each other then," he adds with a smile on his face.`
```

> 			`	"Honest work always bears fruits." His mother, Taepip, picks up, looking at her son, "That's what we always told him after we adopted him."`

Reviewer: Not sure how Taepip's interjection here has much to do with Benjamin's previous line.

Author: ~~you changed it so now it doesnt, but yeah I didnt notice~~
```suggestion
			`	"And now look at you, a starship captain! Honest work always bears fruits." His mother, Taepip, tells you, looking at her son, "That's what we always told him."`
```

			choice
				`	"I was partly-adopted as well."`
			`	They look a bit confused, so you clarify, "My mother was human, but my father was Hai... It took me some time to understand it wasn't my original father. He was loyal to the stars."`
			`	Benjamin answers softly, "Who knows? Maybe he is still out there, somewhere." His mother then speaks up, "If you want, you could come to our home and have a chat. And some pickled acorns, too," she adds with a bright smile.`
			choice
				`	"Sure."`
				`	"Sorry, I have other pressing matters."`
					goto end
			`	You spend a long evening with them, talking about your experience in human space and what you did here before going. "You are sort of like your father," Yeerto comments, before telling you about the local news here.`

> 			`	You almost feel sad when it is already time to go back to your ship.`

Reviewer: ```suggestion
			`	It begins to get dark, and you realize that it may be time to head back to your ship.`
```

			choice
				`	(Leave.)`
					decline
				`	(Approach them.)`
					to display
						has "hai space start"
					goto haistart
				`	"That's your family?"`
					to display
						not "hai space start"
			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
			choice
				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`
				`	"No problem, I hope you enjoyed the trip."`
			label end
			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`
				decline
			label haistart

> 			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then says, "We heard about your travel agency, even went there at some point when we were thinking of relocating to human space. It ended up being a touch too expensive, though, so Benjamin arranged for a merchant to bring him on the way."`

Author: ```suggestion
			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then says, "We heard about your travel agency, even went there at some point when we were thinking of going to human space. It ended up being a touch too expensive, though, so Benjamin arranged for a merchant to bring him on the way."`
```

			`	"Funny we almost found each other then," he adds with a smile on his face.`
			`	"And now look at you, a starship captain! Honest work always bears fruits," his mother Taepip tells you, before looking back at her son. "That's what we always told him."`
			`	She softly places her hand on her son's shoulder before continuing, "If you want, you could come to our home and have a chat. And some pickled acorns, too," she adds with a bright smile.`
			choice
				`	"Sure."`
				`	"Sorry, I have other pressing matters."`
					goto end

> 			`	You spend a long evening with them, talking about your experience in human space and what you did here before going. Yeerto then tells you about the local news here.`

Reviewer: The last sentence here feels a little abrupt. I'd add something on at the end.
```suggestion
			`	You spend a long evening with them, talking about your experience in human space and what you did here before going. Yeerto then tells you about the local news here, and you catch up with what's happened on your home planet since you left.`
```

			choice
				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`
				`	"No problem, I hope you enjoyed the trip."`
			label end
			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`
				decline
			label haistart
			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then says, "We heard about your travel agency, even went there at some point when we were thinking of relocating to human space. It ended up being a touch too expensive, though, so Benjamin arranged for a merchant to bring him on the way."`
			`	"Funny we almost found each other then," he adds with a smile on his face.`
			`	"And now look at you, a starship captain! Honest work always bears fruits," his mother Taepip tells you, before looking back at her son. "That's what we always told him."`

> 			choice
> 				`	"I was partly-adopted as well."`
> 			`	They look a bit confused, so you clarify, "My mother was human, but my father was Hai... It took me some time to understand it wasn't my original father. He was loyal to the stars."`
> 			`	Benjamin answers softly, "Who knows? Maybe he is still out there, somewhere." His mother then speaks up, "If you want, you could come to our home and have a chat. And some pickled acorns, too," she adds with a bright smile.`
> 			choice
> 				`	"Sure."`
> 				`	"Sorry, I have other pressing matters."`
> 					goto end
> 			`	You spend a long evening with them, talking about your experience in human space and what you did here before going. "You are sort of like your father," Yeerto comments, before telling you about the local news here.`

Author: ```suggestion
			`	She softly places her hand on her son's shoulder before continuing, "If you want, you could come to our home and have a chat. And some pickled acorns, too," she adds with a bright smile.`
			choice
				`	"Sure."`
				`	"Sorry, I have other pressing matters."`
					goto end
			`	You spend a long evening with them, talking about your experience in human space and what you did here before going. "You are sort of like your father," Yeerto comments, before telling you about the local news here.`
```

> 			`	It begins to get dark, and after a final goodbye you head back to your ship.`

Author: I wanted to have an explanation for parts of your story in missions but ig I'll remove it

			choice
				`	(Leave.)`
					decline
				`	(Approach them.)`
					to display
						has "hai space start"
					goto haistart
				`	"That's your family?"`
					to display
						not "hai space start"
			`	The man looks at you. "I'm adopted," he says. One of the Hai notices your confusion, and chitters. "Ah, you have not seen a human-Hai family, have you? Perhaps it is strange, but we are happy together nonetheless. Thank you for transporting our son safely."`
			choice
				`	"I didn't mean to be rude, I just didn't know that Hai adopted humans here."`
				`	"No problem, I hope you enjoyed the trip."`
			label end
			`	After thanking you once again, they walk off. As they leave, you overhear Benjamin talk to his family about his trip. "They had so many kinds of food there, it was delicious! Although I did get homesick for pickled acorns..."`
				decline
			label haistart

> 			`	"Of course. You did bring me all the way back here, after all. These are my parents, Yeerto and Taepip." His father Yeerto then says, "We heard about your travel agency, even went there at some point when we were thinking of going to human space. It ended up being a touch too expensive, though, so Benjamin arranged for a merchant to bring him on the way."`

Reviewer 4: Consistency with the previous choice, and making you and Benjamin's family less inexplicably familiar.
```suggestion
			`	Benjamin notices you approaching, and says, "These are my parents, Yeerto and Taepip." His father Yeerto then says, "We checked the travel agencies here on Greenwater, but it turned out to be much cheaper to get a merchant to bring Benjamin to human space. I guess we learned why that was cheaper the hard way."`
```

> 			`	"Funny we almost found each other then," he adds with a smile on his face.`

Reviewer 4: ```suggestion
			`	 "You said you used to work at a travel agency, right?" Benjamin adds with a smile on his face. "Funny we almost found each other then."`
```

> 			`	"And now look at you, a starship captain! Honest work always bears fruits," his mother Taepip tells you, before looking back at her son. "That's what we always told him."`

Reviewer 4: I'm not sure this line makes sense. Why is she congratulating the player on becoming a starship captain? This is their first time meeting, supposedly.
```suggestion
```

> 			`	She softly places her hand on her son's shoulder before continuing, "If you want, you could come to our home and have a chat. And some pickled acorns, too," she adds with a bright smile.`

Reviewer 4: ```suggestion
			`	Benjamin's mother Taepip softly places her hand on his shoulder before continuing, "If you want, you could come to our home and have a chat. And some pickled acorns, too," she adds with a bright smile.`
```

			choice
				`	"Sure."`
				`	"Sorry, I have other pressing matters."`
					goto end

> 			`	You spend a long evening with them, talking about your experience in human space and what you did here before going. Yeerto then tells you about the local news here, and you catch up with what's happened on your home planet since you left.`

Reviewer 4: ```suggestion
			`	You spend a long evening with them, talking about your upbringing on <planet> and your experiences in human space. Yeerto then tells you about the local news here, and you catch up with what's happened on your home planet since you left.`
```

			label maybe
			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' The Republic is aware that you've made contact with the Hai.`
				goto secret
			label grewup

> 			`	"Yes, you are <first> <last>, right?" He takes a look on your face as a confirmation, and checks his pad, "You may have not been seen or heard of in human space for many years, but you were born here, on a spaceship. The Explorer, if my records are correct." He then looks back at you and store his pad. "Trust me, they are.`

Reviewer 2: ```suggestion
			`	"Yes, you are <first> <last>, right?" He takes the look on your face as confirmation, and checks his pad. "You may not have been seen or heard from in human space for many years, but you were born here, on a spaceship: the 'Explorer', if my records are correct." He then looks back at you and store his pad. "Trust me, they are.`
```

> 			`	"Yes, you are <first> <last>, right?" He takes the look on your face as confirmation, and checks his pad. "You may not have been seen or heard from in human space for many years, but you were born here, on a spaceship: the 'Explorer', if my records are correct." He then looks back at you and store his pad. "Trust me, they are.`

Reviewer 2: ```suggestion
			`	"Yes, you are <first> <last>, right?" He takes the look on your face as confirmation, and checks his pad. "You may not have been seen or heard from in human space for many years, but you were born here, on a spaceship: the 'Explorer', if my records are correct." He then looks back at you and stores his pad. "Trust me, they are.`
```
Sorry; I missed one typo in the previous review.

> 			`	"Whilst we don't know every details, the Republic is aware that you've had contact with the Hai."`

Reviewer 2: ```suggestion
			`	"Whilst we don't know every detail, the Republic is aware that you've had contact with the Hai."`
```

			`	"So what is it that you brought me here for?" you ask.`
			`	The officer folds his hands together. "You've made contact with an alien species located to the north. Is this correct?"`
			choice

> 				`	"I did not exactly 'make contact'. I grew up there."`

Reviewer: ```suggestion
				`	"I didn't exactly 'make contact'. I grew up there."`
```

			`	"So what is it that you brought me here for?" you ask.`
			`	The officer folds his hands together. "You've made contact with an alien species located to the north. Is this correct?"`
			choice

> 				`	"I didn't exactly 'make contact'. I grew up there."`

Author: ig I was just too lazy to add a branch haha

> 				`	"I didn't exactly 'make contact'. I grew up there."`

Author: I'll just change it to him not knowing, he's not navy intelligence or anything
btw I think you said smth about him not being a specific government but it says its a Navy officer

			label maybe
			`	The officer chuckles. "I think the answer you're looking for is 'Yes,' not 'Perhaps.' The Republic is aware that you've made contact with the Hai.`
				goto secret
			label grewup

> 			`	"Yes, you are <first> <last>, right?" He takes the look on your face as confirmation, and checks his pad. "You may not have been seen or heard from in human space for many years, but you were born here, on a spaceship: the 'Explorer', if my records are correct." He then looks back at you and stores his pad. "Trust me, they are.`

Author: idea is you were too young to remember anything from that time but that can change

then the thing is you were told you were born on a spaceship, but never that it was in human space specifically ig

> 			`	"Yes, you are <first> <last>, right?" He takes the look on your face as confirmation, and checks his pad. "You may not have been seen or heard from in human space for many years, but you were born here, on a spaceship: the 'Explorer', if my records are correct." He then looks back at you and stores his pad. "Trust me, they are.`

Author: what I was trying to do with the different mission changes is spread little bitties of lore here and there

I'm all for letting the player decide who they want to be but often that just means you don't know, for me at least.

Being born in hai space would be a bit much, honestly I was even thinking of doing a mission where you meet with your dad
